### PR TITLE
Move state transitions diagram into a note

### LINKED ISSUE #342: Is section 3.4 informative or normative?
Section 3.4 doesn't say whether it's informative or normative.  That means that presumably it's normative, but it's not clear to me what the normative content of it is....

I _assume_ this section is meant to be informative and should be annotated as such.

### PR DESCRIPTION
**Pull request by domenic**
Closes #342.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -881,6 +870,14 @@ <h2 id="state-transitions" class="informative">
             </td>
           </tr>
         </table>
+        <div class="note" id="state-transitions">
+          <p>
+            The internal slot <a>[[\state]]</a> transitions between its various
+            potential values as follows:
+          </p><img alt=
```
Comment (marcoscaceres) [inline index.html:877]: Why not use a figure/figcaption here? All the alt info seems valuable to everyone.

**Review by marcoscaceres — CHANGES_REQUESTED** [review]
@domenic, let me know if you want me to do this (add figure/figcaption). I don't mind.

File: index.html (hunk 1 above)
Comment (domenic) [reply index.html:877]: It's strictly redundant with the image, describing the exact same information.

Comment (domenic) [reply index.html:877]: If you think we should repeat it, then we need to do `alt=""` since then the image is redundant.

Comment (marcoscaceres) [reply index.html:877]: yeah, the alt is redundant... don't want to repeat it. Sorry for not being clear.

**Review by marcoscaceres — APPROVED** [review]
I'll add fig caption if/when I do another pass of at fixing the markup.

**Pull request merged**